packages/ui/src/components/Checkbox/style.less

```diff
@@ -6,110 +6,54 @@
 @checkbox-border-width: @condo-global-border-width-default;
 @checkbox-border-radius: @condo-global-border-radius-small;
 @checkbox-size: (@condo-global-spacing-24 - (@checkbox-border-width * 2));
-@checkbox-inner-border-radius: @checkbox-border-radius - @checkbox-border-width;
 
 @condo-checkbox-checkmark-top: ((@checkbox-size / 2));
 @condo-checkbox-checkmark-left: ((@checkbox-size / 4) - 0.5);
 @condo-checkbox-checkmark-width: (@checkbox-size / 4) - 1;
 @condo-checkbox-checkmark-height: (@checkbox-size / 2) - 1.5;
 
+@condo-checkbox-focus-outline-width: @condo-global-border-width-default * 2;
+@condo-checkbox-focus-layer-margin: @condo-checkbox-focus-outline-width + @checkbox-border-width + 2px;
 @condo-checkbox-hidden-area: @condo-global-spacing-8;
 
 .condo-checkbox-wrapper {
   align-items: center;
 
+  * {
+    transition: all @condo-global-transition-duration-default;
```

Comment:
Curve? Like ease-in-out or smth...

Reply:
A bit dangerous selector to apply animations on all properties for all sub-element. Can we be more precise there?

Reply:
was added mixin for transition